package.json

```diff
@@ -25,7 +25,7 @@
     "localforage": "1.5.0",
     "marked": "mattermost/marked#5194fc037b35036910c6542b04bb471fe56b27a9",
     "match-at": "0.1.0",
-    "mattermost-redux": "mattermost/mattermost-redux#master",
+    "mattermost-redux": "mattermost/mattermost-redux#plt-7622",
```

Comment:
this is correct?

Reply:
Yes, just waiting on the mattermost-redux PR to merge and then I'll update this to master with the new commit